desc/objectives/normalization.py

```diff
@@ -38,23 +38,17 @@ def get_lowest_mode(basis, coeffs):
 
         scales["R0"] = R00
         scales["a"] = np.sqrt(np.abs(R10 * Z10))
+        scales["Psi"] = abs(thing.Psi)
+        scales["n"] = 1e19
         scales["A"] = np.pi * scales["a"] ** 2
         scales["V"] = 2 * np.pi * scales["R0"] * scales["A"]
-        scales["B_T"] = abs(thing.Psi) / scales["A"]
-        iota_avg = np.mean(np.abs(thing.get_profile("iota")(np.linspace(0, 1, 20))))
-        if np.isclose(iota_avg, 0):
-            scales["B_P"] = scales["B_T"]
-        else:
-            scales["B_P"] = scales["B_T"] * iota_avg
-        scales["B"] = np.sqrt(scales["B_T"] ** 2 + scales["B_P"] ** 2)
-        scales["I"] = scales["B_P"] * 2 * np.pi / mu_0
+        scales["B"] = scales["Psi"] / scales["A"] * np.sqrt(2)
+        scales["I"] = scales["B"] * 2 * np.pi / mu_0
         scales["p"] = scales["B"] ** 2 / (2 * mu_0)
         scales["W"] = scales["p"] * scales["V"]
```

Comment:
If `p` is the peak pressure on-axis, this will be a huge over-estimate of the total plasma thermal energy. We will need to divide by some correction factor.

Reply:
It's mean to be the total energy (thermal + magnetic)

Reply:
(using p ~ B^2)

Reply:
Well p is almost never proportional to B^2, it is usually 2 orders of magnitude less...

It would be nice to have a normalization for thermal energy, because that's what I'm using in #1168

Reply:
Feel free to add it, this one here is meant to be the scaling factor for the `Energy` objective so its proportional to `B^2`, not the thermal pressure